typedjsonrpc/registry.py

```diff
@@ -29,6 +30,14 @@
 __all__ = ["Registry"]
 
 
+def _get_default_logger():
+    logger = logging.getLogger(__name__)
```

Comment:
Usually you configure a logger in one place and then use it in others (by name).

If you set the log level to INFO here, if someone decides to modify the logging configuration before this, then that configuration is wiped out.